.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,29 @@
-<!--
-Thanks for submitting a pull request to ESLint. Before continuing, please be sure you've read over our guidelines:
-http://eslint.org/docs/developer-guide/contributing/pull-requests
+**What is the purpose of this pull request? (put an "X" next to item)**
 
-Specifically, all pull requests containing code require an **accepted** issue (documentation-only pull requests do not require an issue). If this pull request contains code and there isn't yet an issue explaining why you're submitting this pull request, please stop and open a new issue first.
+```
+[ ] Documentation update
+[ ] Bug fix ([template](https://github.com/eslint/eslint/blob/master/templates/bug-report.md))
+[ ] New rule ([template](https://github.com/eslint/eslint/blob/master/templates/rule-proposal.md))
+[ ] Changes an existing rule ([template](https://github.com/eslint/eslint/blob/master/templates/rule-change-proposal.md))
+[ ] Add autofixing to a rule
+[ ] Add a CLI option
+[ ] Add something to the core
+[ ] Other, please explain:
+```
 
-Please answer all questions below.
--->
+*If the item you've check above has a template, please paste the template questions here and answer them.*
 
-**What issue does this pull request address?**
 
 
+**Please check each item to ensure your pull request is ready:**
+
+- [ ] I've read the [pull request guide](http://eslint.org/docs/developer-guide/contributing/pull-requests)
```

Comment:
Do we want to phrase this as a list of checkboxes? This feels a lot like "Yes, I've read terms and conditions" callouts. Maybe we should just mention that to make successful pull request you should do the following things?